examples/consul/README.md

```diff
@@ -0,0 +1,37 @@
+# consul
```

Comment:
Move `consul/` to `examples/dubbo/consul` or `examples/jsonrpc/consul` according using protocol.

What do you about?

Reply:
@fangyincheng Yes, I think that adding protocol to file path will be better.